### ISSUE TITLE
Support onClick

### ISSUE THREAD

**Issue by ShuaiYuan87** [CookPete/react-player#452]
Hi @CookPete thanks for the amazing repo! I'm glad I found your repo. Something I was wondering if you could add support is: I'd like a onClick prop in [here](https://github.com/CookPete/react-player/blob/master/src/props.js) The reason is then I can get the event myself and notify my server. I cannot use the callback as my server may trigger play/pause to the videos so I need to distinguish if the call is initiated from a click or the server. Is it something you can help to support? Thanks in advance!

**Comment by CookPete** [comment]
Hey @ShuaiYuan87 does this not already work? Any non-supported prop should be [passed through](https://github.com/CookPete/react-player/blob/488f82da421ac9a27c2e52137767d19cc7f47a3e/src/ReactPlayer.js#L106) to the wrapper element, so `onClick` should work.

Ah I guess it depends what URL you are trying to play, as some players use iframes, which will consume any click events.

I'm not sure exactly how your implementation works, but can you not listen to the `onPlay` event and determine on the app side whether or not it was a result of a server trigger? You can assume if it wasn't a server trigger then it must have been a user click.

**Comment by ShuaiYuan87** [comment]
yeah it seems it was captured by the video frame and not passing to the wrapper.

Good point on the app side! will try that. Thanks for the quick reply!

**Comment by ShivangiKhandelwal27** [comment]
I  am running to a similar situation where in I have passed a wrapper to my react-player which is a React Component returning children wrapped in a div. Handler/function passed to OnClick inside react player or even that Div (which is in my wrapper component) isn't getting called.
I want to full screen my youtube video on a single click anywhere on the video.
 Checked on browser via devtools, the click event is attached but not getting called.
Anyone can help regarding the same ?
